sale_order_line_packaging_qty/models/sale_order_line.py

```diff
@@ -35,16 +35,18 @@ def _compute_product_packaging_qty(self):
                 )
             else:
                 product_qty = sol.product_uom_qty
+            # provide default values in case of missing product
```

Comment:
```suggestion
            # Provide default values in case of missing product.
            # Odoo does not prevent you to modify product packaging
            # that are already linked to a sale order.
            # If by chance the product is removed (yes, you can do it)
            # this computation will be broken w/out this default.
```